src/ResourceManager/LogicApp/AzureRM.LogicApp.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.5.0'
+ModuleVersion = '3.0.0'
```

Comment:
@pankajsn we will take care of handling versions in PS modules. So can you please revert this change.